lumigator/python/mzai/backend/backend/config_templates.py

```diff
@@ -1,5 +1,6 @@
-seq2seq_template = """{{
-    "name": "{experiment_name}/{experiment_id}",
+# Evaluation templates
```

Comment:
Note that we now have two sets of config templates, one for evaluation and one for inference

Reply:
will eval and batch inference take the same parameters? The reason I split them out is that for ground truth inference (or others) we don't need the eval metrics.

Reply:
> eval and batch inference take the same parameters

I wouldn't expect it. Let's see what you all think - I would imagine batch inference to just need the parameters the model needs to generate content (and this could be just predictions to be tested or it could be "ground-truth" from a reference model). The evaluation would take it from there, separately (metric, params the metric needs, references to the data it has to compare/measure).


> reason I split them out is that for ground truth inference (or others) we don't need the eval metrics

Completely agree.

Reply:
I agree on splitting jobs, for this reason I did not see evaluation needing a list of model configurations (as an inference job will be used to run inference for that down the line).
But I think it's ok to keep it like this (separate configs) for now, and then when eval only does evals we can remove redundant eval model_configs